docs/src/examples/components/Tree/Types/TreeExampleExperiment.shorthand.tsx

```diff
@@ -0,0 +1,166 @@
+import * as React from 'react'
```

Comment:
this example is way to complex to have it here; I'd suggest making it an actual prototype under `Prototypes/` section